src/orb.yml

```diff
@@ -211,7 +211,6 @@ executors:
         default: medium
     docker:
       - image: circleci/node:<<parameters.version>>
-    working_directory: ~/react-native
```

Comment:
Removing from `executor` level since we added a parameter at the job level with a default value which will always override the `executor` value for this option - `working_directory`.